src/collections/_documentation/platforms/react-native/sourcemaps.md

```diff
@@ -44,3 +44,7 @@ Sentry.setRelease(RELEASE_NAME)
 `DISTRIBUTION_NAME`:
 
 : This is the version code or build ID depending on your platform. So for instance just set this to whatever is set in your _Info.plist_ or what your Gradle setup generates (For example, `52`).
+
+```js
+Sentry.setDist(`dist`);
```

Comment:
We mention in the previous paragraph that an example is `52`, should this match?

Reply:
The example directly above this section for setting releases uses: Sentry.setRelease(RELEASE_NAME)`

Should we match that and use
Sentry.setDist(DISTRIBUTION_NAME)

instead?

Reply:
O yeah @mbauer404 's suggestion makes more sense since you would never hardcode a `52`

Reply:
Great suggestion, @mbauer404 ! Let me know if there are other changes we should address. If not, feel free to approve. ;)